WordPress/src/main/java/org/wordpress/android/ui/compose/components/card/UnelevatedCard.kt

```diff
@@ -6,21 +6,23 @@ import androidx.compose.material.Card
 import androidx.compose.material.MaterialTheme
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
 import androidx.compose.ui.unit.dp
 
 @Composable
 fun UnelevatedCard(
     modifier: Modifier = Modifier,
-    content: @Composable () -> Unit
+    content: @Composable () -> Unit,
 ) {
+    val shape = RoundedCornerShape(10.dp)
     Card(
-        modifier = modifier,
+        modifier = Modifier.clip(shape).then(modifier),
```

Comment:
This adds rounded corners to a click ripple effect.

Reply:
TIL :)